libkbfs/disk_block_cache_helper_test.go

```diff
@@ -0,0 +1,71 @@
+package libkbfs
+
+import (
+	"testing"
+	"time"
+
+	"github.com/keybase/kbfs/kbfscodec"
+	"github.com/keybase/kbfs/tlf"
+	"github.com/stretchr/testify/require"
+)
+
+func makeTime(data []byte) time.Time {
+	var t time.Time
+	err := t.UnmarshalBinary(data)
+	if err != nil {
+		panic(err)
+	}
+	return t
+}
+
+var testLRUTime = makeTime([]byte{
+	// Version.
```

Comment:
Did this to try to get a consistent value across OSs (and to make the encoding more obvious)